### PR TITLE
Sirius mainnet node release

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Documentation updates for Sirius mainnet node release.

## Changes

Release notes, update downloads topic, update run macos node topic.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [ ] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
Update release note and downloads after testnet reset.

**Comment by dg-concordium** [comment]
I think it is only missing the packages for nodes for ubuntu and docker along with checksums.

File: source/mainnet/net/installation/downloads.rst [hunk 1]
```diff
@@ -34,66 +34,71 @@ The Concordium Mobile Wallet has been verified by NowSecure.
...
-   - SHA256 checksum of the download: :substitution-code:`|client-linux-checksum|`
-   - :ref:`Verification instructions <verification-client-linux>`
+      - SHA256 checksum of the download: :substitution-code:`|client-linux-checksum|`
+      - :ref:`Verification instructions <verification-client-linux>`
 
--  `Download the Mainnet Concordium Client for macOS <https://distribution.concordium.software/tools/macos/signed/concordium-client_3.0.4-0.zip>`_.
+-  `Download the Mainnet Concordium Client for macOS <https://distribution.concordium.software/tools/macos/signed/concordium-client_4.0.3-0.zip>`_
 
--  `Download the Mainnet Concordium Client for Windows <https://distribution.concordium.software/tools/windows/signed/concordium-client_3.0.4-0.exe>`_
+-  `Download the Mainnet Concordium Client for Windows <https://distribution.concordium.software/tools/windows/concordium-client_4.0.3-0.exe>`_
```
Comment (abizjak) [inline source/mainnet/net/installation/downloads.rst:86]: We should be using a signed client for windows as well.

File: source/mainnet/net/installation/downloads.rst [hunk 2]
```diff
@@ -103,41 +108,41 @@ Ubuntu
 
 To run a node on a server with Ubuntu, you need a Debian package.
 
-   - `Download the Mainnet Debian package <https://distribution.mainnet.concordium.software/deb/concordium-mainnet-node_3.0.2_amd64.deb>`_
+   - `Download the Mainnet Debian package <https://distribution.mainnet.concordium.com/deb/concordium-testnet-node_4.1.1_amd64.deb>`_
```
Comment (abizjak) [inline source/mainnet/net/installation/downloads.rst:111]:
```suggestion
   - `Download the Mainnet Debian package <https://distribution.mainnet.concordium.software/deb/concordium-mainnet-node_4.1.1_amd64.deb>`_
```

Please make sure to update the hash of the package as well.

File: source/mainnet/net/installation/downloads.rst [hunk 3]
```diff
@@ -103,41 +108,41 @@ Ubuntu
...
-Docker-Linux
+Linux-Docker
 ------------
 
 .. _concordium-docker-package-download:
 
 Download the full suite for running a node on Linux using Docker. The suite contains Concordium Node, Concordium Client and cargo-concordium.
 
-   - `Download the Mainnet suite for Linux <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-3.0.2-0-mainnet.tar.gz>`_
+   - `Download the Mainnet suite for Linux <https://distribution.mainnet.concordium.com/tools/linux/concordium-software-linux-4.1.1-1-testnet.tar.gz>`_
```
Comment (abizjak) [inline source/mainnet/net/installation/downloads.rst:125]:
```suggestion
   - `Download the Mainnet suite for Linux <https://distribution.mainnet.concordium.software/tools/linux/concordium-software-linux-4.1.1-1-mainnet.tar.gz>`_
```

File: source/mainnet/net/installation/downloads.rst [hunk 4]
```diff
@@ -103,41 +108,41 @@ Ubuntu
...
 To learn how to run a node with Docker, see :ref:`Run a node with Docker <run-a-node>`.
 
 Windows
 -------
 
-To run a node on Windows, you need a Windows Installer package.
+To run a node on Windows, you need a Windows Installer package. **Please be aware that you should backup your configuration, as the installer will overwrite the current configuration with a standard configuration.**
 
-   - `Download the Mainnet Windows Installer package <https://distribution.concordium.software/windows/Signed/Node-3.0.2.msi>`_
+   - `Download the Mainnet Windows Installer package <https://distribution.concordium.software/windows/Signed/Node-4.1.1.msi>`_
```
Comment (abizjak) [inline source/mainnet/net/installation/downloads.rst:136]: To be checked.

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 5]
```diff
@@ -4,11 +4,69 @@
...
+June 15, 2022
+
+Concordium Node 4.1.1
+---------------------
+
+Concordium Node 4.1.1 introduces new functionality to support delegation to baker pools or passive delegation, and a new version Smart Contracts.
+
+Note that when the protocol update happens on June 23, 2022 that the cool-down period for reducing baker stake or stopping baking increases from one week to two weeks. If you reduce your stake or stop baking BEFORE the protocol update takes effect, the cool-down remains one week.
+
+V1 smart contracts includes the following key features:
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes-mainnet.rst:19]:
```suggestion
V1 smart contracts include the following key features:
```

**Review by abizjak — CHANGES_REQUESTED** [review]

**Review by abizjak — APPROVED** [review]

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 6]
```diff
@@ -4,11 +4,69 @@
...
+=================
+
+June 15, 2022
+
+Concordium Node 4.1.1
+---------------------
+
+Concordium Node 4.1.1 introduces new functionality to support delegation to baker pools or passive delegation, and a new version Smart Contracts.
+
+Note that when the protocol update happens on June 23, 2022 that the cool-down period for reducing baker stake or stopping baking increases from one week to two weeks. If you reduce your stake or stop baking BEFORE the protocol update takes effect, the cool-down remains one week.
```
Comment (mh-concordium) [inline source/mainnet/net/resources/release-notes-mainnet.rst:17]: Baker (pool owner) cooldown is set for 3 weeks. So it increases from one week to 3 weeks.
Delegator cooldown is set to 2 weeks.

Comment (dg-concordium) [reply source/mainnet/net/resources/release-notes-mainnet.rst]: Fixed.

**Pull request merged**